### PR TITLE
Add QVEX Lynepad

### PR DESCRIPTION
**Pull request by seflm**
## Description

Add QVEX Lynepad to VIA

## QMK Pull Request 

Lynepad support PR merged into QMK master on 11th July 2023.
https://github.com/qmk/qmk_firmware/pull/21070

## Checklist

- [x] The VIA support for this keyboard is **MERGED** in QMK master already **(MANDATORY)**
- [x] The VIA definition follows the guide here: https://caniusevia.com/docs/layouts
- [x] I have a V3 JSON version for this keyboard definition.**(MANDATORY)**
- [x] I have tested this keyboard definition using VIA's "Design" tab.
- [x] I have tested this keyboard definition with firmware on a device.
- [ ] I have assigned alpha keys and modifier keys with the correct colors.
- [x] The Vendor ID is not `0xFEED`

### CODE REVIEW

**Comment by seflm** [comment]
Hello, can someone (@Xelus22 ) please clarify why the build failed? It seems it expected a V2 definition according to the following build errors

```
Error: v2 definition invalid: ../src/qvex/lynepad2/lynepad2.json
Error: KeyboardDefinitionV2 must have required property 'lighting', KeyboardDefinitionV2 must NOT have additional properties, KeyboardDefinitionV2 must NOT have additional properties
```

I thought V2 is deprecated, and only V3 is allowed for PRs.

Thank you

**Comment by seflm** [comment]
Am I supposed to insert the definition into the v3 folder instead of src?

**Comment by seflm** [comment]
Accidentally put a v3 definition in the src folder instead of the v3 folder, fixed.

File: v3/qvex/lynepad2/lynepad2.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+            ["0,0","0,1","0,2","0,3"],
+            ["1,0","1,1","1,2","1,3"],
+            ["2,0","2,1","2,2"],
+            [{"y":-0.75,"x":4.5},"3,0"],
+            [{"x":3.5},"3,2","4,0\n\n\n\n\n\n\n\n\ne0","3,3"],
+            [{"x":4.5},"3,1"],
+            [{"y":0.5,"x":3.25},"4,1\n\n\n\n\n\n\n\n\ne1"]
+        ]
+    },
+    "menus": ["qmk_backlight","qmk_rgblight"],
```
Comment (Xelus22) [inline v3/qvex/lynepad2/lynepad2.json:23]: I believe you want this instead

`[qmk_backlight_rgblight](https://www.caniusevia.com/docs/built_in_menus#qmk_backlight_rgblight)`

https://www.caniusevia.com/docs/custom_ui

**Review by Xelus22 — CHANGES_REQUESTED** [review]

File: v3/qvex/lynepad2/lynepad2.json (hunk 1 above)
Comment (seflm) [reply v3/qvex/lynepad2/lynepad2.json]: Thanks for pointing this out. I ended up creating a custom menu that is more suitable for what the Lynepad can do.

**Comment by seflm** [comment]
Ended up creating a custom menu that is more suitable for what the Lynepad can do.

**Review by Xelus22 — APPROVED** [review]

**Pull request merged**